src/classlibnative/bcltype/system.cpp

```diff
@@ -343,13 +343,6 @@ INT32 QCALLTYPE SystemNative::GetProcessorCount()
         processorCount = systemInfo.dwNumberOfProcessors;
     }
 
-#ifdef FEATURE_PAL
-    uint32_t cpuLimit;
-
-    if (PAL_GetCpuLimit(&cpuLimit) && cpuLimit < (uint32_t)processorCount)
-        processorCount = cpuLimit;
-#endif
-
     END_QCALL;
```

Comment:
Do we want to remove this? Doesn't that change `Environment.ProcessorCount` to no longer take into account --cpus?

Reply:
Yes, that's the goal of the change.